Polyhedron/demo/Polyhedron/Viewer.cpp

```diff
@@ -403,6 +409,12 @@ void Viewer::mousePressEvent(QMouseEvent* event)
   }
   else if(!event->modifiers()
           && event->button() == Qt::LeftButton
+          && d->z_is_pressed)
+  {
+      d->scene->zoomToPosition(event->pos(), this);
+  }
```

Comment:
It is documented (see the screenshot), and I use it every day! That is one of the most useful shortcut of QGLViewer.

![screenshot3](https://cloud.githubusercontent.com/assets/5746675/25190826/c4b18564-252d-11e7-9233-b3a980aefa94.png)